### DISCUSSION TITLE
Overrides not working

### DISCUSSION THREAD

**Discussion by nicolas377** [github-linguist/linguist#5715] [Q&A]
Hey there! I maintain a repo [here](https://github.com/gpsystem/geofs-types) that is just a bunch of .d.ts files. I know that .d.ts files are considered vendored, and I'm trying to undo that, but it doesn't seem to be working. Could someone help me out with an override to unmark a file extension as vendored (or documentation, or whatever)?

I know I've probably glossed past the answer a couple times, sorry about this!

**Comment by Nixinova** [comment, marked as answer]
Add an `=false` to the end. `**/*.d.ts linguist-vendored=false`

**Reply by lildude** [reply]
Or use `-linguist-vendored` - note the `-` prefix - it's the second example in the [vendored section of the override doc](https://github.com/github/linguist/blob/master/docs/overrides.md#vendored-code) 😉.
